### DISCUSSION TITLE
Bluetooth type

### DISCUSSION THREAD

**Discussion by gmptrr98** [iRobotEducation/create3_docs#162] [Questions] [answered]
Hello, 
I'm considering using Create 3 to reach a position which is communicated to the robot by the presence of either a beacon or tag, so by using BLE. 
What I wanted to ask is what's the Bluetooth version on Create 3? Can a beacon or tag ping a position to Create 3 directly or should I install a receiver of some kind? And once once I get the position, could I use the ROS command "Navigate To Position" to reach it?
Thanks again, sorry for the many questions but I'm currently waiting on the delivery of my Create 3 robot and can't wait to test all this things myself!

**Comment by shamlian** [comment, marked as answer]
Create 3 has a radio capable of Bluetooth 5.0 but in normal operation only uses it for Bluetooth Low Energy when in [BLE mode](https://iroboteducation.github.io/create3_docs/hw/adapter/#usbble-toggle). I'm not sure exactly what it is you'd like to do, but I'm not sure that the robot is designed to do what you're expecting.
